refinery/core/views.py

```diff
@@ -698,6 +699,14 @@ class NodeViewSet(viewsets.ModelViewSet):
     # permission_classes = (IsAuthenticated,)
 
 
+class EventViewSet(viewsets.ModelViewSet):
+    """API endpoint that allows Events to be viewed"""
+    queryset = Event.objects.all()
+    serializer_class = EventSerializer
+    # Don't think I need a lookup_field?
```

Comment:
Yeah I think the default is the `Object.pk`, and at the moment I don't think we're trying to have a detail view for `Events`

Reply:
Comment on which events are returned? Per user?